lib/reports/ReportCollisionDirectory.js

```diff
@@ -69,21 +69,112 @@ class ReportCollisionDirectory extends ReportBaseCrash {
      * TODO: this violates the "no shared references" invariant in `ReportBase#transformData`.
      * We should address that eventually, but there isn't time right now.
      */
-    const rawCollisions = rawData.collisions;
-    const collisions = rawCollisions.map(
+    const collisions = rawData.collisions.map(
       ReportCollisionDirectory.getCollisionDirectoryEntry,
     );
     const collisionSummary = { ...rawData.collisionSummary };
-    return { collisions, collisionSummary, rawCollisions };
+    return { collisions, collisionSummary };
   }
 
-  generateCsv(location, { rawCollisions }) {
-    if (rawCollisions.length === 0) {
-      return { columns: [], rows: [] };
-    }
-    const columns = Object.keys(rawCollisions[0])
-      .map(key => ({ key, header: key }));
-    return { columns, rows: rawCollisions };
+  getCsvDriverCells({
```

Comment:
right now, we have no tests on these reports at all.  since these access `collision_factors`, we'd either have to a) put them in our database-backed tests, or b) mock out `collision_factors` fetching to make these unit tests.

another possible approach is that collision reports - especially tabulation reports! - have a bunch of logic that could be unit tested independently of the full end-to-end pipeline.

Reply:
refactored this a bit to make unit testing of that separable logic easier in the future, but we should tackle this when we have more time.